cpp/src/table/table.cpp

```diff
@@ -28,12 +28,13 @@ namespace cudf {
 
 table::table(std::vector<gdf_column*> const& cols) : _columns{cols} {
   CUDF_EXPECTS(nullptr != cols[0], "Null input column");
-  this->_num_rows = cols[0]->size;
+  gdf_size_type num_rows = cols[0]->size;
 
-  std::for_each(_columns.begin(), _columns.end(), [this](gdf_column* col) {
-    CUDF_EXPECTS(nullptr != col, "Null input column");
-    CUDF_EXPECTS(_num_rows == col->size, "Column size mismatch");
-  });
+  std::for_each(_columns.begin(), _columns.end(),
+                [this, num_rows](gdf_column* col) {
```

Comment:
I prefer the former formatting -- it makes a `std::for_each` read more like a for loop, which is what it is.

Reply:
I agree with you, it is just a lot less overhead to use whatever formatting VSCode gives me rather than trying to manually maintain formatting.

Reply:
Isn't that configurable? :)